docs-kits/kits/Industry Core Kit/Software Development View/part_aspect-model-overview.mdx

```diff
@@ -32,4 +32,6 @@ In the following section, all aspect models that are part of Industry Core are d
 | | JustInSequencePart | 3.0.0 | | Industry Core |
 | | SingleLevelBomAsBuilt | 3.0.0 | | Industry Core |
 | | SingleLevelUsageAsBuilt | 3.0.0 | | Industry Core |
-| | TractionBatteryCode | 1.0.0 | | Traceability |
+| | TractionBatteryCode | 2.0.0 | | Traceability |
+| | SoftwareInformation | 1.0.0 | | Traceability |
+| | CertificateSigningRequests | 1.0.0 | | Traceability |
```

Comment:
The section says "_In the following section, all aspect models that are part of Industry Core are documented_", but these two aspect models are currently not seem to be covered by the Traceability standards, nor by the Industry Core standards.

Reply:
@jSchuetz88 those two aspect models are not mentioned in the corresponding use case standard document, since there are not interest to standardize these models yet. New aspect models do not have to be standardized automatically after publishing - am i wrong? 

To standardize the two aspect models will be the task of the Traceability Expert Group an may be done earliest for R24.12. 

@mhellmeier

Reply:
@jSchuetz88 since the section is used by the Traceability KIT and Industry Core KIT, i've changed the sentence to be capable for both KITs:
 _"The following section documents all Aspect models that are part of or relate to an Industry Core digital twin."_

@eckardg FYI

Reply:
Please change line 23 as follows:
"The following section shows an overview of all aspect models that are currently attached to part type or part instance twins by Catena-X use cases".